.github/ISSUE_TEMPLATE/epic-issue-template.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Epic Issue Template
+about: Issue template used for Epics
+title: ''
+labels: Epic
+assignees: ''
+
+---
+
+## Desired Feature
+A clear and concise description of what the feature is and what it does.
+
+## Actual/Current Behavior
+A clear and concise description of what is currently happening.
+
+## Steps to Implement
+Add a list of the steps required to implement this feature
+
+## Epic Checklist
+Add a list of issues related to this epic
```

Comment:
perhaps this should include an example checklist?

Reply:
@thomascking Thats a good idea. Something like:

- [ ] Issue #997
- [ ] Issue #998
- [ ] Issue #999

Or maybe just have the issue number:
- [ ] #997
- [ ] #998
- [ ] #999

?